README.md

```diff
@@ -10,10 +10,10 @@ Users can use this feature in two configurations.
 
 In the cluster-aware connection load balancing, connections are distributed across all the tservers in the cluster, irrespective of their placements.
 
-To enable the cluster-aware connection load balancing, provide the parameter `load_balance` set to true as `load_balance=true` in the connection url or the connection string (DSN style).
+To enable the cluster-aware connection load balancing, provide the parameter `load_balance` set to true or any as `load_balance=any` in the connection url or the connection string (DSN style).
 
 ```
-"postgres://username:password@localhost:5433/database_name?load_balance=true"
```

Comment:
We should keep value `true` in some examples to let users know even that is supported. even if it is mentioned explicitly.